app/code/Magento/Integration/README.md

```diff
@@ -1,6 +1,97 @@
-# Integration
+# Magento_Integration module
 
-**Integration** enables third-party services to call the Web API by using access tokens.
+This module enables third-party services to call the Web API by using access tokens.
 It provides an admin UI that enables manual creation of integrations. Extensions can also provide a configuration
 file so that an integration can be automatically pre-configured. The module also contains the data
 model for request and access token management.
+
+## Installation
+
+Before installing this module, note that the Magento_Integration is dependent on the following modules:
+- `Magento_Store`
+- `Magento_User`
+- `Magento_Security`
+
+The following modules depend on this module:
+- `Magento_Analytics`
+- `Magento_Webapi`
+
+The Magento_Integration module creates the following tables in the database:
```

Comment:
I think we should not mention any schema / data reverting as this module should not be disabled or removed, because a lot of modules depends of it. I added info about it. What do you think?